Analysis/Tasks/PWGHF/taskCorrelationDDbar.cxx

```diff
@@ -63,26 +63,34 @@ const double binMinSparse[4] = {-o2::constants::math::PI / 2.,-6.,0.,0.;  //is t
-     {"hCorrel2DPtIntSignalRegion", "D,Dbar candidates signal region;#it{#varphi}^{Dbar}-#it{#varphi}^{D};#it{#eta}^{Dbar}-#it{#eta}^{D};entries", {HistType::kTH2F, {{32, -o2::constants::math::PI / 2., 3. * o2::constants::math::PI / 2.}, {200, -10., 10.}}}},
-     {"hCorrel2DVsPtSignalRegion", "D,Dbar candidates signal region;#it{#varphi}^{Dbar}-#it{#varphi}^{D};#it{#eta}^{Dbar}-#it{#eta}^{D};#it{p}_{T}^{D};#it{p}_{T}^{Dbar};entries", {HistType::kTHnSparseD, {{32, -o2::constants::math::PI / 2., 3. * o2::constants::math::PI / 2.}, {120, -6., 6.}, {10, 0., 10.}, {10, 0., 10.}}}}, //note: axes 3 and 4 (the pT) are updated in the init()
-     {"hDeltaPtDDbarSignalRegion", "D,Dbar candidates signal region;#it{p}_{T}^{Dbar}-#it{p}_{T}^{D};entries", {HistType::kTH1F, {{144, -36., 36.}}}},
-     {"hDeltaPtMaxMinSignalRegion", "D,Dbar candidates signal region;#it{p}_{T}^{max}-#it{p}_{T}^{min};entries", {HistType::kTH1F, {{72, 0., 36.}}}},
-     {"hDeltaEtaPtIntSidebands", "D,Dbar candidates sidebands;#it{#eta}^{Dbar}-#it{#eta}^{D};entries", {HistType::kTH1F, {{200, -10., 10.}}}},
-     {"hDeltaPhiPtIntSidebands", "D,Dbar candidates sidebands;#it{#varphi}^{Dbar}-#it{#varphi}^{D};entries", {HistType::kTH1F, {{32, -o2::constants::math::PI / 2., 3. * o2::constants::math::PI / 2.}}}},
-     {"hCorrel2DPtIntSidebands", "D,Dbar candidates sidebands;#it{#varphi}^{Dbar}-#it{#varphi}^{D};#it{#eta}^{Dbar}-#it{#eta}^{D};entries", {HistType::kTH2F, {{32, -o2::constants::math::PI / 2., 3. * o2::constants::math::PI / 2.}, {200, -10., 10.}}}},
-     {"hCorrel2DVsPtSidebands", "D,Dbar candidates sidebands;#it{#varphi}^{Dbar}-#it{#varphi}^{D};#it{#eta}^{Dbar}-#it{#eta}^{D};#it{p}_{T}^{D};#it{p}_{T}^{Dbar};entries", {HistType::kTHnSparseD, {{32, -o2::constants::math::PI / 2., 3. * o2::constants::math::PI / 2.}, {120, -6., 6.}, {10, 0., 10.}, {10, 0., 10.}}}}, //note: axes 3 and 4 (the pT) are updated in the init()
-     {"hDeltaPtDDbarSidebands", "D,Dbar candidates sidebands;#it{p}_{T}^{Dbar}-#it{p}_{T}^{D};entries", {HistType::kTH1F, {{144, -36., 36.}}}},
-     {"hDeltaPtMaxMinSidebands", "D,Dbar candidates sidebands;#it{p}_{T}^{max}-#it{p}_{T}^{min};entries", {HistType::kTH1F, {{72, 0., 36.}}}}}};
+    {{"hMass2DCorrelationPairs", "D,Dbar candidates 2D;inv. mass D (GeV/#it{c}^{2});inv. mass Dbar (GeV/#it{c}^{2});" + stringPtD + stringPtDbar + ";entries", {HistType::kTHnSparseD, {{200, 1.6, 2.1}, {200, 1.6, 2.1}, {10, 0., 10.}, {10, 0., 10.}}}}, //note: axes 3 and 4 (the pT) are updated in the init()
+     {"hDeltaEtaPtIntSignalRegion", "D,Dbar candidates signal region;" + stringDeltaEta + ";entries", {HistType::kTH1F, {{200, -10., 10.}}}},
+     {"hDeltaPhiPtIntSignalRegion", "D,Dbar candidates signal region;" + stringDeltaPhi + ";entries", {HistType::kTH1F, {{32, -o2::constants::math::PI / 2., 3. * o2::constants::math::PI / 2.}}}},
```

Comment:
Looks much better. I think it would even be worth it defining separate strings for:
* "D,Dbar candidates"
* "D,Dbar particles - MC gen"
* "signal region"
* "sidebands"

Also please check for missing `;` axis label delimiter at some places.

Reply:
I also introduced those further labels, and moved the ';' inside the strings definition to simplify the string composition, it should be ok now